tests/models/test_model.py

```diff
@@ -39,8 +40,6 @@ def test_api_attrs():
     assert hasattr(dm, "_API")
     assert "asdf" in dm.api
     assert dm.api["asdf"] == "yeah"
-    assert "region" in dm.api
```

Comment:
Can update to check for Region class. I see in the current implementation we are not initializing with an empty gdf anymore, good to reference in the v1 changelog

Reply:
I'll add the check that we have a region here, but I don't understand the rest of your comment. What would you like to be referenced in the chagnelog?